docker/README.md

```diff
@@ -0,0 +1,137 @@
+
+```bash
+docker run --rm -it -v "$(pwd):/workdir/external" \
+    ghcr.io/gammasim/simtools-prod:latest \
+    simtools-print-array-elements \
+    --array_element_list ./simtools/tests/resources/telescope_positions-North-utm.ecsv \
+    --export corsika --use_corsika_telescope_height \
+    --output_path /workdir/external/
+```
+
+Output files can be found `./simtools-output/`.
+
+### Building a simtools-prod container
+
+To build a new container locally run:
+
+```bash
+docker build -f Dockerfile-prod  -t simtools-prod .
+```
+
+Building will take a while and the image is large (~1.4 GB). For using images build on your own, replace in all examples `ghcr.io/gammasim/simtools-prod:latest` by the local image name `simtools-prod`.
+
+## Container for simtools Developers
+
+Provide a container for testing and development of simtools, including:
+
+- CORSIKA and sim\_telarray
+- packages required by simtools (from pyproject.toml)
+
+The container does not include the simtools code, which should be cloned in a separate directory (see below).
+
+### Run a container using the prepared Docker image available from repository
+
+Packages are available from the [simtools container repository](https://github.com/orgs/gammasim/packages?repo_name=simtools).
+
+**Preparation:**
+
+Create a new directory for your development and clone simtools into a subdirectory:
+
+```bash
+mkdir -p simtools-dev && simtools-dev
+git clone git@github.com:gammasim/simtools.git
+```
+
+To download and run a prepared container in bash:
+
+```bash
+docker run --rm -it -v "$(pwd)/:/workdir/external" ghcr.io/gammasim/simtools-dev:latest bash -c "$(cat ./simtools/docker/entrypoint.sh) && bash"
+```
+
+This additionally executes the `entrypoint.sh` script (e.g., for pip install or to set the database environment).
+
+Remember you need to `docker login` to the GitHub package repository with a personal token in order to download an image (follow [these instructions](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)).
+
+### Build a new developers container locally
+
+To build a new container locally run:
+
+```bash
+docker build -f Dockerfile-dev -t simtools-dev .
+```
+
+Use the docker container in the same way as above, replacing `ghcr.io/gammasim/simtools-dev:latest` by `simtools-dev`.
+
+## Container for simulation software CORSIKA / sim\_telarray
+
+Provide a container including the following the CORSIKA and sim\_telarray simulation software packages.
+
+### Download from simtools container repository
+
+Packages are available from the [simtools container repository](https://github.com/gammasim/containers/pkgs/container/simtools-simtel).
+
+To download and run a prepared container in bash:
+
+```bash
+docker run --rm -it -v "$(pwd)/external:/workdir/external" ghcr.io/gammasim/simtools-simtelarray:latest bash
+```
+
+## Build a new container locally
+
+To build a new container locally run:
```

Comment:
Please include already here a note (and not after the build command) that at this point we strictly need `corsika7.7_simtelarray.tar.gz` inside the `simtools/docker` directory for the following commands to work.

Reply:
yes, it is clear. my comment was more on the position in the text where the note appears. in my opinion it should come before running the command, otherwise one can run the command and only later realize that the file was needed.